x-pack/solutions/observability/plugins/apm/ftr_e2e/cypress/e2e/dependencies/dependencies.cy.ts

```diff
@@ -150,3 +151,26 @@ describe('Dependencies with high volume of data', () => {
     cy.contains('nav', 'Page 1 of 100');
   });
 });
+
+describe('Dependencies page with no dependencies', () => {
+  before(() => {
+    synthtrace.clean(); // ensure no dependencies data
+    synthtrace.index(
+      generateDataWithoutDependencies({
```

Comment:
Instead of generating data without any dependencies, you can simply go to the dependencies page and select a time range where you know no data exists.

wdyt of this approach?